Gemfile

```diff
@@ -89,7 +89,7 @@ gem "will_paginate", "~> 3.0"
 gem "xmlrpc", "~> 0.3"
 
 # For creating many records, quickly
-gem "fast_inserter", "~> 0.1"
+gem "fast_inserter", "~> 2.0"
```

Comment:
I checked the changelog and this seems fine. it seems to have test coverage as well :)